### ISSUE TITLE
problem about acc

### ISSUE THREAD

**Issue by wq1148927746** [lucastabelini/PolyLaneNet#45]
Hello. Thank you for your contribution. I want to know what is the difference between the two Acc in the following picture? What do they mean? The result of my operation is about 88%.

![image](https://user-images.githubusercontent.com/47206549/112713997-23a37e80-8f13-11eb-9441-a07388a7fb3d.png)
![image](https://user-images.githubusercontent.com/47206549/112714009-2a31f600-8f13-11eb-9d2a-9602cd7d9351.png)

**Comment by lucastabelini** [comment]
Table I shows the accuracy metric (described in Equation 7) for some lane detection methods on the test set of the TuSimple dataset. Table II shows the accuracy metric for PolyLaneNet using different polynomial degrees in the validation set of the TuSimple dataset.

**Comment by wq1148927746** [comment]
> Table I shows the accuracy metric (described in Equation 7) for some lane detection methods on the test set of the TuSimple dataset. Table II shows the accuracy metric for PolyLaneNet using different polynomial degrees in the validation set of the TuSimple dataset.

Why is there so much difference between the accuracy on the test set and the validation set?

**Comment by lucastabelini** [comment]
Apart from the sets being different, the only other difference is that the model tested in the test set was trained using the training+validaton sets, while the one tested on the validation set was trained using only the training set.
